### PR TITLE
Updates

### LINKED ISSUE #19: failed to select a version for `gtk`
Trying to build master (ca736637064a8ba87a9d1d3713d05b5e52fa33d2) results in the following error:

```
error: failed to select a version for `gtk`.
    ... required by package `gtk-test v0.2.0 (/home/jonas/dev/gtk-test)`
versions that meet the requirements `*` are: 0.5.0

the package `gtk-test` depends on `gtk`, with features: `v3_10` but `gtk` does not have these features.
```

This might have been caused by the git dependencies getting out of date.

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Fixes #19.

cc @EPashkin @sdroege

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -21,7 +21,7 @@ gdk = { git = "https://github.com/gtk-rs/gdk" }
 gtk = { git = "https://github.com/gtk-rs/gtk" }
 
 [features]
-default = ["gtk/v3_10"]
+default = ["gtk/v3_16"]
```
Comment (sdroege) [inline Cargo.toml]: Why do you suddenly need `v3_16` here?

Comment (GuillaumeGomez) [reply Cargo.toml]: I just replaced. But I guess it's pretty useless in fact... I'll remove it.

Comment (sdroege) [reply Cargo.toml]: You should only add such things if you're actually requiring new API from 3.16 :)

Comment (sdroege) [reply Cargo.toml]: Good to go after that change

File: .travis.yml [hunk 2]
```diff
@@ -31,9 +31,9 @@ before_script:
...
 script:
   - rustc --version
-  - if [ "$TRAVIS_RUST_VERSION" == "nightly" ]; then
-    rustup component add clippy;
-    cargo clippy --release;
-    fi
+  # - if [ "$TRAVIS_RUST_VERSION" == "nightly" ]; then
+  #   rustup component add clippy;
+  #   cargo clippy --release;
+  #   fi
```
Comment (sdroege) [inline .travis.yml]: What's wrong with them?

Comment (GuillaumeGomez) [reply .travis.yml]: Look at travis logs, you'll see what dark magic is. :)

Comment (sdroege) [reply .travis.yml]: It's installed by default now or not? There's no separate component for it.

Comment (GuillaumeGomez) [reply .travis.yml]: I couldn't make it work and at this point, it's just annoying.

Comment (sdroege) [reply .travis.yml]: Ah no. The problem is: https://rust-lang-nursery.github.io/rust-toolstate/

Comment (GuillaumeGomez) [reply .travis.yml]: Ah indeed. I remove the last commit and then I merge.

**Pull request merged**